modules/services/betterlockscreen.nix

```
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.betterlockscreen;
+
+in {
+  meta.maintainers = with maintainers; [ sebtm ];
+
+  options = {
+    services.betterlockscreen = {
+      enable = mkEnableOption
+        "the betterlockscreen systemd-service for sleep/suspend.";
```

the `.` is added automatically by `mkEnableOption`. Also, "system-service" sounds weird. I think "systemd service" is better.

```suggestion
        "the betterlockscreen systemd service for sleep/suspend";
```

---

I just changed the text to something more meaningful and removed the dot thanks for pointing out :)